diff --git a/bin/dispatch-conf b/bin/dispatch-conf
index 16c9a5d..c658b4a 100755
--- a/bin/dispatch-conf
+++ b/bin/dispatch-conf
@@ -156,7 +156,7 @@ class dispatch:
                 continue

             basename = "*"
-            find_opts = ["-name", ".*", "-type", "d", "-prune", "-o"]
+            find_opts = ["-name", ".*", "-type", "d", "-follow", "-prune", "-o"]
             if not os.path.isdir(path):
                 path, basename = os.path.split(path)
                 find_opts = ["-maxdepth", "1"]
